test/bdd/db/import/linking.feature

```diff
@@ -310,3 +310,21 @@ Feature: Linking of places
          | object     | name+_place_name  |
          | R1         | LabelPlace |
 
+
+    Scenario: Linked places expand default language names
+        Given the grid
+            | 1 |   | 2 |
+            |   | 9 |   |
+            | 4 |   | 3 |
+        Given the places
+            | osm | class    | type   | name+name | name+name:en | name+name:ru | geometry |
+            | N9  | place    | city           | Popayán | Popayán | Попаян | 9 |
```

Comment:
You can even leave out the English and Russian names. They are not relevant for the example. (But they also do no harm.)

Can you please make sure the table columns are properly aligned? Makes it easier to read the tests.